pkg/ui/src/FormFields/TextField/TextField.svelte

```diff
@@ -20,11 +21,13 @@
 	export let variant: TextFieldSize = TextFieldSize.Base;
 	export let error = false;
 	export let value = "";
+	export let onHelpTextClick = () => {};
 	$: error = error;
 
 	const labelBaseClasses = ["block", "text-sm", "font-medium", "text-gray-700"];
 	const helpTextBaseClasses = ["ml-[2px]", "text-sm", "min-h-[20px]"];
 	const inputBaseClasses = ["block", "w-full", "border-0", "text-sm", "focus:outline-0", "focus:ring-0", variant];
+	const clickableHelpText = onHelpTextClick ? "cursor-pointer underline" : "";
```

Comment:
This will always evaluate to true because you've assigned default `() => {}` - even this empty function is truthy.